inlong-sort/sort-connectors/src/main/java/org/apache/inlong/sort/flink/hive/HiveWriter.java

```diff
@@ -68,9 +74,24 @@
 
     private transient List<HivePartition> newPartitions;
 
+    private static UserGroupInformation proxyUgi;
+
     public HiveWriter(Configuration configuration, long dataFlowId, HiveSinkInfo hiveSinkInfo) {
         this.configuration = checkNotNull(configuration);
         this.dataFlowId = dataFlowId;
+        UserGroupInformation realUgi = null;
+        try {
+            realUgi = UserGroupInformation.getLoginUser();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Pls, use the LOG framework instead of print.

Reply:
done